src/server/scripts/Outland/BlackTemple/boss_illidari_council.cpp

```diff
@@ -512,13 +521,25 @@ class spell_illidari_council_balance_of_power_aura : public AuraScript
 
     void CalculateAmount(AuraEffect const* /*aurEff*/, int32& amount, bool& /*canBeRecalculated*/)
     {
-        // Set absorbtion amount to unlimited (no absorb)
+        // Set absorption amount to unlimited (no absorb)
         amount = -1;
     }
 
+    void Absorb(AuraEffect* /*aurEff*/, DamageInfo& dmgInfo, uint32& /*absorbAmount*/)
+    {
+        Unit* councilMember = GetTarget();
+        InstanceScript* instance = councilMember->GetInstanceScript();
```

Comment:
we need to nullcheck instance here, if the spell is cast otuside instance it will crash

Reply:
Corrected.